app/scripts/app.module.js

```diff
@@ -85,10 +91,20 @@
           data: { requireLogin: false }
         });
 
-    }).run(function ($rootScope, $state, $location, OpenmrsRestService) {
 
-      $rootScope.$on('$stateChangeStart', function (event, toState, toParams) {
+    }).run(function ($rootScope, $state, $location, OpenmrsRestService, OpenmrsSettings, EtlRestServicesSettings) {
```

Comment:
Is it possible to use annotation or $inject in this case? We don't want surprises in case we decide to uglify our code :smile: